.github/workflows/kube-score-checker.yaml

```diff
@@ -22,5 +22,4 @@ jobs:
         - name: Run kube-score
           run: |
               cd charts/cytomine
-              helm template cytomine . | kube-score score -
-        
+              helm template cytomine . | kube-score score - --ignore-test pod-probes --ignore-test container-security-context-readonlyrootfilesystem --ignore-test container-security-context-user-group-id
```

Comment:
For the ones that are plain wrong (e.g. I disagree with the probe test and believe I do so from with an informed opinion), you can add annotations to the respective object (e.g. like https://github.com/NBISweden/gc-bp-helm/blob/main/charts/grand-challenge/templates/web-deployment.yaml#L5) to ignore that test. That seems better to me than changing the workflow to exclude the tests (particularly since I guess it applies to all objects then).

(To do so, it's good to be able to explain why it's wrong, just ping me if you feel the need to discuss.)

I strongly suspect a few of those will be needed/very strongly desired for deployment for BigPicture but require some work to fix them up to working (feel free to signal if you want help with that.

Reply:
Thanks, that feels more selective.

And I suspect some of these will be needed when deploying to sc-cluster (and such a good idea that it'll  be worth the work), but we'll cross that bridge et.c.